src/server/nonClassEvent/nonClassEvent.service.ts

```diff
@@ -56,4 +68,16 @@ export class NonClassEventService {
       .orderBy('p.area', 'ASC');
     return nonClassEvents.getMany();
   }
+
+  public async createWithNonClassEvents(parent: Partial<NonClassParent>):
```

Comment:
This is missing docs

Reply:
D'oh! So it is. Fixed that in 7b83146d1dd91e2e8df2d0e9065b7b52ebf1bdf7 as well as added some docs for the `find` method above it.